LondonBritishLibrary/orient/BLorient546.xml

```diff
@@ -31,21 +31,165 @@ schematypens="http://relaxng.org/ns/structure/1.0"
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.3.4">
+<locus from="77r"/>
+<title type="complete" xml:lang="gez" ref="LIT1570habeka">Of St John the evangelist</title>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.3.5">
+<locus from="86r"/>
+<title type="complete" xml:lang="gez" ref="LIT3120Anapho">Of John Chrysostom</title>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.3.6">
+<locus from="91r"/>
+<title type="complete" xml:lang="gez" ref="LIT1162Athana">Of Athanasius</title>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.3.7">
+<locus from="101v"/>
+<title type="complete" xml:lang="gez" ref="LIT1347Epipha">Of Epiphanius</title>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.3.8">
+<locus from="107v"/>
+<title type="complete" xml:lang="gez" ref="LIT1103Anapho">Of the 318 Nicene fathers</title>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.3.9">
+<locus from="115v"/>
+<title type="complete" xml:lang="gez" ref="LIT2040Naakkw">Of Gregory of Alexandria <sic/></title>
+<incipit xml:lang="gez"></incipit>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.3.10">
+<locus from="121r"/>
+<title type="complete" xml:lang="gez" ref="LIT3180Anapho">Of Dioscorus</title>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.3.11">
+<locus from="123r"/>
+<title type="complete" xml:lang="gez" ref="LIT2404Tanseu">Of Jacob of Sǝrug</title>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.3.12">
+<locus from="128v"/>
+<title type="complete" xml:lang="gez" ref="LIT1098Anapho">Of Basil of Caesarea</title>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.3.13">
+<locus from="137r"/>
+<title type="complete" xml:lang="gez" ref="LIT2038Naakkw">Of Gregory of Armenia</title>
+<incipit xml:lang="gez">ነአኵተከ፡ እግዚኦ፡ ንሴብሐከ፡ ወንዌድሰከ፡ ቡሩክ፡ ስምከ፡ ወንባርከከ፡ ንሕነኒ፡ ውዱስ፡ ስምከ፡</incipit>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.3.14">
+<locus from="140r"/>
+<title type="complete" xml:lang="gez" ref="LIT1569habeka">Of Cyril of Alexandria</title>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i4">
+<locus from="146r"/>
+<title type="complete" ref="LIT4942Sarawita"/>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i5">
+<locus from="147v"/>
+<title type="complete" xml:lang="en">Prayers</title>
+<textLang mainLang="gez"></textLang>
+
+<msItem xml:id="ms_i5.1">
+<locus from="147v"/>
+<title type="complete" ref="LIT2252salota">Prayer of the incense</title>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i5.2">
+<locus from="152v"/>
+<title type="incomplete" xml:lang="en" ref="LIT2167Parali">Prayer of Jeremiah</title>
+<textLang mainLang="gez"></textLang>
+<note>It is a fragment of Paralipomena of Jeremiah.</note>
+</msItem>
+
```

Comment:
For supplication encoding, please, consider: https://betamasaheft.eu/Guidelines/?q=supplication&id=ColophSupplTit
I am not quite sure, which would be the best variant here, maybe as an explicit of the previous text?

Reply:
Do you want to consider the supplication as a separate msItem?

Reply:
By my judgement, this is too insignificant a text for a separate msItem. If you think otherwise, let me know and I will create a work record.

Reply:
But now you consider it as a separate msItem (ms_i5.3), do not you? :) I do not mean creation of a work record, but the way of encoding the supplication we have documented in the link I have copied above. If you scroll down to the examples 5, 6, etc., you will see the encoding of a supplication as an incipit/explicit could work here. In this case you won't need a separate msItem

Reply:
OK, I think I'm getting closer ;)